<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_opened_file">
      <property name="text">
       <string>opened file label</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="main_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>421</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="__state_format">
           <property name="text">
            <string>Save State Format</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="state_format_combo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="__bg_color_toggle_label">
           <property name="text">
            <string>Override Background Color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="bg_color_toggle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="__bg_color_override_label">
           <property name="text">
            <string>Override Color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="__img_output_dir_label">
           <property name="text">
            <string>Image Output Directory</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="output_directory_line">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="output_select_button">
             <property name="toolTip">
              <string>Select Directory</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/folder-open.svg</normaloff>:/icons/folder-open.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="ColorButton" name="bg_color_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="default_config_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>93</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Restore Defaults</string>
        </property>
       </widget>
      </widget>
      <widget class="TabPalette" name="tab_palette">
       <attribute name="title">
        <string>Palette</string>
       </attribute>
      </widget>
      <widget class="TabVram" name="tab_vram">
       <attribute name="title">
        <string>VRAM</string>
       </attribute>
      </widget>
      <widget class="TabRaw" name="tab_raw">
       <attribute name="title">
        <string>RAW Tiles</string>
       </attribute>
      </widget>
      <widget class="TabTileMap" name="tab_tilemap">
       <attribute name="title">
        <string>Tile Maps</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_hw_sprites">
       <attribute name="title">
        <string>Hardware Sprites</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_folder"/>
   <addaction name="action_open_file"/>
   <addaction name="action_previous_file"/>
   <addaction name="action_next_file"/>
   <addaction name="separator"/>
   <addaction name="action_lock_palette"/>
  </widget>
  <action name="action_open_folder">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/folder-open.svg</normaloff>:/icons/folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="toolTip">
    <string>Select save state folder</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_previous_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/chevron-left.svg</normaloff>:/icons/chevron-left.svg</iconset>
   </property>
   <property name="text">
    <string>Previous File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_next_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/chevron-right.svg</normaloff>:/icons/chevron-right.svg</iconset>
   </property>
   <property name="text">
    <string>Next File</string>
   </property>
   <property name="toolTip">
    <string>Next File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_lock_palette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/palette.svg</normaloff>
     <normalon>:/icons/world.svg</normalon>:/icons/palette.svg</iconset>
   </property>
   <property name="text">
    <string>Swap Palette</string>
   </property>
   <property name="toolTip">
    <string>Swap Palette</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/file.svg</normaloff>:/icons/file.svg</iconset>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
   <property name="toolTip">
    <string>Select Save State File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>..ui.colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TabPalette</class>
   <extends>QWidget</extends>
   <header>..ui.tabpalette.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabVram</class>
   <extends>QWidget</extends>
   <header>..ui.tabvram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabRaw</class>
   <extends>QWidget</extends>
   <header>..ui.tabraw.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabTileMap</class>
   <extends>QWidget</extends>
   <header>..ui.tabtilemap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>main_tabs</tabstop>
  <tabstop>bg_color_toggle</tabstop>
  <tabstop>bg_color_button</tabstop>
  <tabstop>output_directory_line</tabstop>
  <tabstop>output_select_button</tabstop>
  <tabstop>state_format_combo</tabstop>
  <tabstop>default_config_button</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
